src/nxdoc/nuxeo-server/platform-services/automation/automation-scripting.md

```diff
@@ -320,40 +320,31 @@ If one of your Automation chain or operation contains dashes (`-`) in their name
 
 {{/callout}}
 
-The following code example is bound to a simple action button, renames all selected documents with the value "test" as `dc:title` or displays a warning JSF message (depending on the number of the selected documents).
+The following code example is bound to a simple event (`Document created`), to apply property value inheritance from the parent document, with some logging information messages.
```

Comment:
No need to put _Document created_ in a code block as it doesn't represent any kind of code.